nodes/parachain/src/rpc.rs

```diff
@@ -57,29 +59,24 @@ where
 	C::Api: BlockBuilder<Block>,
 	P: TransactionPool + Sync + Send + 'static,
 {
-	use frame_rpc_system::{FullSystem, SystemApi};
-	use pallet_transaction_payment_rpc::{TransactionPayment, TransactionPaymentApi};
+	use frame_rpc_system::{System, SystemApiServer};
+	use pallet_transaction_payment_rpc::{TransactionPayment, TransactionPaymentApiServer};
 
-	let mut io = jsonrpc_core::IoHandler::default();
+	let mut module = RpcExtension::new(());
 	let FullDeps {
 		client,
 		pool,
 		deny_unsafe,
 	} = deps;
 
-	io.extend_with(SystemApi::to_delegate(FullSystem::new(
```

Comment:
The `FullSystem` is the important part here. We use(d) this for the `NextNonce` call. It was refactored to `SystemRpc` in [Substrate #8783](https://github.com/paritytech/substrate/pull/8783/files#diff-079bdf5b7d7db5208076eef2e7d1aec9ff00ccbb3c947ddda949ee9ceb0a6347R83) and is now named [`System`](https://github.com/paritytech/substrate/blob/polkadot-v0.9.23/utils/frame/rpc/system/src/lib.rs#L81).